paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/cdc/Expression.java

```diff
@@ -37,7 +38,7 @@
 public interface Expression extends Serializable {
 
     List<String> SUPPORTED_EXPRESSION =
-            Arrays.asList("year", "month", "day", "hour", "substring", "truncate");
+            Arrays.asList("year", "month", "day", "hour", "dateFormat", "substring", "truncate");
```

Comment:
maybe `date_format`?

Reply:
@JingsongLi  I resubmitted the code ， could you help to review this pull request?